src/main/java/org/prebid/server/bidder/adkernel/AdkernelBidder.java

```diff
@@ -79,9 +78,9 @@ private void processImp(Imp imp, Map<ExtImpAdkernel, List<Imp>> pubToImps) {
     }
 
     private static void validateImp(Imp imp) {
-        if (imp.getBanner() == null && imp.getVideo() == null) {
-            throw new PreBidException(String.format(
-                    "Invalid imp id=%s. Expected imp.banner or imp.video", imp.getId()));
+        if (imp.getBanner() == null && imp.getVideo() == null && imp.getXNative() == null) {
```

Comment:
Should we update `media-types` in adkernel.yaml?

Reply:
at the current time not